cmd/scheduler/metrics.go

```diff
@@ -75,73 +76,74 @@ func (cc ClusterManagerCollector) Collect(ch chan<- prometheus.Metric) {
 	nodevGPUMemoryLimitDesc := prometheus.NewDesc(
 		"GPUDeviceMemoryLimit",
 		"Device memory limit for a certain GPU",
-		[]string{"nodeid", "deviceuuid"}, nil,
+		[]string{"nodeid", "deviceuuid", "gpuid"}, nil,
 	)
```

Comment:
deviceuuid should be gpuid. deviceuuid is also the device id obtained through nvml.